pkg/option/config.go

```diff
@@ -440,6 +440,9 @@ const (
 	// RoutingMode is the name of the option to choose between native routing and tunneling mode
 	RoutingMode = "routing-mode"
 
+	// SubnetTopologyFilePath is the path to the file containing the subnet topology
+	SubnetTopologyFilePath = "subnet-topology-file-path"
```

Comment:
bikeshedding: Is there a more descriptive name that indicates that this is used to impact the routing mode? Maybe something like `native-routing-subnet-topology` or similar?